### DISCUSSION TITLE
Skewed distributions of the inter-individual random effects

### DISCUSSION THREAD

**Discussion by Percival24** [nlmixr2/nlmixr2#114] [Q&A] [answered]
Hi! 
I am running a pk model combining data coming from multiple groups.
I am using est = 'focei' + outeropt = "nlminb". 
There are random effects on 5 parameters. 
I expected each parameter to see the estimated random effects being distributed around 0. 
But it is not the case... 
For some parameters, the distribution of random effects is skewed and clearly above or below zero... 
Does this mystery make sens to anybody?
Best regards, 
Percival

**Comment by mattfidler** [comment, marked as answer]
Hi @Percival24 

Sometimes this does happen with non-linear mixed effect models.  You can try a few things to get them to work better:

- rerun the model with the estimates or tweak the estimates a bit to get a better fit.  Model piping is useful here.
- Try some coding some skewed random effects distribution [described here](https://www.page-meeting.org/pdf_assets/880-semiparametric%20distributions%20with%20estimated%20shape%20parameters%20implementation%20and%20evaluation.pdf)

**Comment by mattfidler** [comment]
So, the results could be better, but it is not too much different than what could be observed elsewhere.

**Reply by mattfidler** [reply]
I converted this to a discussion, since it really isn't an issue.

**Comment by Percival24** [comment]
Many thanks @mattfidler!  It is really helpful :)
